plugins/3-laravel.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+test -f artisan || return 1
+
+laravel_console() {
+  run php artisan tinker
+}
+
+laravel_server() {
+  run php artisan serve
+}
+
+laravel_test() {
+  run vendor/bin/phpunit
```

Comment:
Is `phpunit` used outside of laravel? Is there a different config file that it looks for? In other words, is there ever not an `artisan` file but someone might still want `scrappy test` to run `phpunit`?

Reply:
Yes, `phpunit` is a separate package from Laravel, but it's the default testing framework from Laravel. 

Would it be better to create a separate plugin file for `phpunit` that only runs the test command?